src/libaktualizr/storage/invstorage.h

```diff
@@ -139,6 +139,7 @@ class INvStorage {
   virtual void saveReportEvent(const Json::Value& json_value) = 0;
   virtual bool loadReportEvents(Json::Value* report_array, int64_t* id_max, int limit = -1) const = 0;
   virtual void deleteReportEvents(int64_t id_max) = 0;
+  virtual void truncateReportEvents(int64_t max_events_count) = 0;
```

Comment:
I wonder if it is possible to achieve what we need without adding a new method to the interface?